modules/consentManagement.js

```diff
@@ -374,9 +556,20 @@ export function setConsentConfig(config) {
       utils.logError(`consentManagement config with cmpApi: 'static' did not specify consentData. No consents will be available to adapters.`);
     }
   }
-  if (!addedConsentHook) {
+
+  if (consentModule === 'usp' && !addedConsentHookUSP) {
+    $$PREBID_GLOBAL$$.requestBids.before(requestCcpaBidsHook, 50);
+    addedConsentHookUSP = true;
+  }
+
+  if (!addedConsentHook && consentModule === 'gdpr') {
     $$PREBID_GLOBAL$$.requestBids.before(requestBidsHook, 50);
+    addedConsentHook = true;
   }
-  addedConsentHook = true;
 }
-config.getConfig('consentManagement', config => setConsentConfig(config.consentManagement));
+
+config.getConfig('consentManagement', config => {
+  const consentManagement = { ...config.consentManagement };
+  const consentChecks = consentManagement.consentAPIs ? new Set([...consentManagement.consentAPIs]) : new Set([]);
```

Comment:
we need to be backward compatible, so even if `consentAPIs` does not exist, we still need to support `gdpr`. 
 So does `gdpr` get set somewhere?

Reply:
I actually showed you this earlier. If `config.consentManagement.cmpApi`is set, I was adding `GDPR` into the set. I took it out because you mentioned just relying on the `consentAPIs` array.

Reply:
may have been some confusion.  But i don't think `cmpApi` is required because it defaults to 'iab'.  just want to make sure it defaults to gdpr if `consentAPIs` is not included.